cmake/EthCompilerSettings.cmake

```diff
@@ -71,7 +71,7 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MA
 	add_compile_options(-fPIC)
 
 	# Configuration-specific compiler settings.
-	set(CMAKE_CXX_FLAGS_DEBUG          "-Og -g -DETH_DEBUG")
```

Comment:
Oh, sorry, that sneaked in. It actually does make a difference for my current compiler (it does not know that option), but I did not investigate further.